### DISCUSSION TITLE
Add synchronous operations

### DISCUSSION THREAD

**Discussion by jaimefuhr** [Jericho/ZoomNet#199] [General]
Currently using the library in a web application.   I need to wait for the meeting ID to be returned, for example, when creating a Scheduled Meeting.

Can a synchronous method for CreateScheduledMeeting be added, or what is the best practice?

Thanks!

**Comment by Jericho** [comment]
The best practice it to await the result of the asynchronous method like so:
```csharp
var newScheduledMeeting = await client.Meetings.CreateScheduledMeetingAsync(myUser.Id, "ZoomNet Integration Testing: scheduled meeting", "The agenda", start, duration, TimeZones.UTC, "p@ss!w0rd", settings, trackingFields, null, cancellationToken).ConfigureAwait(false);
```
